functions.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+function getDb() : PDO {
+    $db = new PDO('mysql:host=db; dbname=damian-collection', 'root', 'password');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    return $db;
+}
+
+function retrieveAlbums(PDO $db) : array {
+    $query = $db->prepare("SELECT `name`, `artist`, `release-year`, `genre`, `record-label`, `image-link` FROM `albums`");
+    $query->execute();
+    return $query->fetchAll();
+}
+
+function displayAlbums(array $albums) : string {
+
+    if($albums === []) {
+        $displayalbums = 'ERROR ERROR ERROR';
```

Comment:
haha not the most helpful error message - perhaps 'cannot find any albums, please try again later.' might be a little more suitable?

Reply:
done